syntaxes/qc.tmLanguage.json

```diff
@@ -21,23 +21,41 @@
         {
 			"include": "source.kv#stringQuoted"
 		},
-        {
-			"include": "source.kv#stringUnquoted"
-		}
+		{
+            "include": "#stringUnquoted"
+        }
```

Comment:
What is the purpose of this? You copy pasted stringUnquoted from source.kv without doing any notable changes I can find

Reply:
It may look the previous commit didn't had any change bacause it was a simple removal of the explicit `$` and double backlash the `-` as range switch

However we got from this:
![2022-06-10-202502_1920x1059_scrot](https://user-images.githubusercontent.com/9941414/173162110-59e36013-1de2-4204-88b5-7ccf508b3454.png)

to this:
![2022-06-15-155526_1920x983_scrot](https://user-images.githubusercontent.com/9941414/173904168-4eaca5e6-4800-4a4e-9c77-2a0e6d076543.png)

Reply:
I see now but I don't like this solution as the string is termimated once a variable is found and then another string begins after the variable.

A better solution would be to nest the variable inside of the string. Use the begin and end captures here